audio-worklet/basic/bit-crusher.html

```diff
@@ -20,25 +33,28 @@
         const context = new AudioContext();
         let oscillator = new OscillatorNode(context);
         let bitCrusher = new AudioWorkletNode(context, 'bit-crusher');
+
+        oscillator.type = 'sawtooth';
+        oscillator.frequency.value = 5000;
```

Comment:
Why the change in oscillator type?  Better sounding?

Reply:
Yes, it really shows off the nice effect of bit crusher. More higher frequency content, more crushing sounds.